data/successors/successor side missions.txt

```diff
@@ -0,0 +1,488 @@
+			`	One of the other ones pipes up, cheerfully adding, "We don't know anything!" before the others give them a dark look.`
+			choice
+				`	"And it's so colorful too! Are you sure it's not a paint of some sort?"`
+					goto colorful
+				`	"Fascinating. Carry on."`
+					goto fooled
+
+			label colorful
+			`	"Oh, that's for ... ah ... safety. Yes, 'tis very dangerous to touch if thou'r'nt trained, so 'tis bright in color as a warning." None of the three are wearing any sort of protective equipment that you can see, and by now most of the group have changed color to what you might characterize as a mirthful shade of pink. This is something which one or two are clearly trying to suppress, their skin shifting in gradients in an effort to maintain their previous dull purple hue. One of the others is trying to hide a bag filled with more cans of "hull solution" behind their arms.`
+			choice
+				`	"Well, if you're working on behalf of the port, then surely I can ask you to remove it as well."`
+					goto called
+				`	"Nice try, but you and I both know what's going on here."`
+					goto called
+				`	"I think my ship's hull has been well enough repaired. It might be best for you to attend to whatever other vessels are on your list."`
+					goto complicit
+				`	"Thanks for answering my questions. Please carry on."`
+					goto fooled
+
+			label called
+			`	The trio seem rather deflated at hearing this, but after quite a bit of milling amongst themselves eventually relent and agree to remove the solution so long as you don't inform the port authority about their actions. While they put good effort into removing it---or at least act the part---they can't seem to scrub away the most dried-on parts, and you eventually let them go after it becomes apparent that some of the paint has permanently bonded to the hull.`
+				decline
+
+			label complicit
+			`	The trio brighten up at hearing this, and after some milling amongst themselves eventually glide off into the spaceport with their cans of solution in tow, though not before leaving one with you for "emergency repairs," though considering how terrible it smells it's not certain if you could ever stand trying to apply it.`
+				decline
+
+			label ignored
+			`	Upon returning from the spaceport some time later, you find that an area on the side of the <ship> has been covered in a thick, colorful solution with an incredibly noxious smell so potent you can hardly bear to approach it. The pattern forms some words in the Successor script, and while the translation device struggles to interpret the non-standard glyphs, it eventually offers a translation along the lines of "Ryiaa was here (and friends)." The defacement takes several hours for the spaceport crews to sufficiently scour off, but a small degree of discoloration remains permanently bonded to the hull plating for the foreseeable future.`
```

Comment:
Looks like I missed something in #[19](https://github.com/Daeridanii1/successors/pull/19).
```suggestion
			`	Upon returning from the spaceport some time later, you find that an area on the side of the <ship> has been covered in a thick, colorful solution with an incredibly noxious smell so potent you can hardly bear to approach it. The pattern forms some words in the Successor script, and while the translation device struggles to interpret the non-standard glyphs, it eventually offers a translation along the lines of "Ryiaa was here (and friends.)" The defacement takes several hours for the spaceport crews to sufficiently scour off, but a small degree of discoloration remains permanently bonded to the hull plating for the foreseeable future.`
```